cpp/autosar/src/rules/A0-1-3/UnusedLocalFunction.ql

```diff
@@ -19,6 +19,44 @@ import codingstandards.cpp.autosar
 import codingstandards.cpp.DynamicCallGraph
 import codingstandards.cpp.deadcode.UnusedFunctions
 
+/** Checks if a function call exists to the function
+ * passed in the arguments.
+ */
+predicate isCalled(Function unusedFunction) {
+  exists (FunctionCall f | unusedFunction.getACallToThisFunction() = f)
+}
+
+/** Checks if an overloaded function of
+ * the function passed in the arguments, is called.
+ */
+predicate overloadedFunctionIsCalled(Function unusedFunction) {
+  exists (Function f | f = unusedFunction.getAnOverload*() and isCalled(f))
```

Comment:
I don't believe you need a `*` here - `getAnOverload` should identify all overloads.
```suggestion
  exists (Function f | f = unusedFunction.getAnOverload() and isCalled(f))
```